File: Algorithms/Sorting/Bubble Sort/BubbleSort.java
import java.util.Scanner;

public class BubbleSort{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int array[] = new int[5];
        for(int i = 0; i < array.length; i++){
            array[i] = sc.nextInt();
        }

        System.out.print("Before: ");
        for(int i : array){
            System.out.printf("%d,", i);
        }

        for(int i = 0; i < array.length; i++){
            int done = 1; 
            for(int j = i+1; j < array.length; j++){
                if (array[i]>array[j]){
                    int temp = array[i];
                    array[i] = array[j];
                    array[j] = temp;
                    done = 0;
                }
            }  
            if(done==1){
                break;
            }
        }

        System.out.print("After: ");
        for(int i : array){
            System.out.printf("%d,", i);
        }
        
    }
}

File: Algorithms/Sorting/Bubble Sort/BubbleSort.cpp
#include<iostream>
using namespace std;

int main(){
    int array[5] = {0,0,0,0,0};

    for(int i = 0; i<(sizeof(array)/sizeof(array[0])); i++){
        cin >> array[i]; 
    }

    cout << "Before: ";
    for(int i: array){
        cout << i << ',';
    }

    for(int i = 0; i<(sizeof(array)/sizeof(array[0])); i++){
        int done = 1;
        for(int j = i; j<(sizeof(array)/sizeof(array[0])); j++){
            if(array[i]>array[j]){
                int temp = array[i];
                array[i] = array[j];
                array[j] = temp;
                done = 0;
            }
        }
        if(done){
            break;
        }
    }

    cout << "After: ";
    for(int i: array){
        cout << i << ',';
    }
}